modules/data_release/help/data_release.md

```diff
@@ -0,0 +1,3 @@
+**Data Release**
+
+The Data Release Module can be used to easily distribute packa    ged data releases of your study. Use the "Upload File" button to upload your file and tag it with a version based on your own version convention. Grant access to any of your use    rs using the "Add Permission" button. These buttons will be visible only if you have the "superuser" permission.
```

Comment:
There are spaces in the middle of some of these words (ie "packa    ged", "use     rs")